pkg/apis/forklift/v1beta1/plan.go

```diff
@@ -101,8 +101,8 @@ func (p *Plan) VSphereUsesEl9VirtV2v() (bool, error) {
 		return false, liberr.New("Cannot analyze plan, destination provider is missing.")
 	}
 
-	if source.Type() != VSphere {
-		return false, nil
+	if source.Type() == VSphere || source.Type() == Ova {
```

Comment:
the first part is problematic - if `source.Type() == VSphere` then we need to check `!p.Spec.Warm && destination.IsHost()`

Reply:
@bkhizgiy I posted https://github.com/kubev2v/forklift/pull/463 to ease extending this function, please take a look

Reply:
Just saw, looks better in as a switch case